### ISSUE TITLE
yet another "Could not connect to VLC... " 

### ISSUE THREAD

**Issue by Telperion87** [nicolasbrailo/VlcFreemote#25]
I've already checked the same issue here https://github.com/nicolasbrailo/VlcFreemote/issues/19.
I've tried to troubleshoot the problem since I'm experiencing the same issue: Looking the incoming connections over port 8080 on the vlc "server" pc, I've seen connections from my phone when I used the browser interface but no incoming connection whatsoever when using vlcfreemote.

So I tried to use another phone and actually vlcFreemote connection worked!
The only difference worth considering between the two phones is the Android version: android 10 not working and android 6 working.

I was wondering if the issues could be caused by the android 10 upgrade, since I've already experienced other important changes and issues in the behavior of the apps after the last version came out for my phone

thanks

**Comment by nicolasbrailo** [comment]
> android 10 not working and android 6 working.

Well, that sucks. And it would very much explain the problem. I have no idea what changed in Android 10 to break this, though, as far as I know normal network permissions should be enough for this app to work. I'm not sure when I'll be able to troubleshoot this issue, unfortunately, as I won't have access to my dev machine for a while...

Thanks for reporting!

**Comment by marmistrz** [comment]
Works on Android 7, doesn't work on Android 9. There's nothing interesting in the logcat:
```
11-25 17:41:01.021   629   629 I ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
11-25 17:41:02.684   629   629 I chatty  : uid=1000(system) /system/bin/surfaceflinger identical 6 lines
11-25 17:41:02.685   629   629 I ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
11-25 17:41:03.067   768   768 W android.hardware.power@1.3-service.pro1: Acquire: called while uninitialized
11-25 17:41:03.175   768   768 I chatty  : uid=1000(system) power@1.3-servi identical 2 lines
11-25 17:41:03.176   768   768 W android.hardware.power@1.3-service.pro1: Acquire: called while uninitialized
11-25 17:41:03.352 20971 20971 I MainActivity: Connecting to server 192.168.0.2:8080
11-25 17:41:03.372 20971 20989 D OpenGLRenderer: endAllActiveAnimators on 0x74cf842f00 (RippleDrawable) with handle 0x74e0a3a720
11-25 17:41:03.389   768   768 W android.hardware.power@1.3-service.pro1: Acquire: called while uninitialized
11-25 17:41:03.391   768   768 I chatty  : uid=1000(system) power@1.3-servi identical 1 line
11-25 17:41:03.490   768   768 W android.hardware.power@1.3-service.pro1: Acquire: called while uninitialized
11-25 17:41:03.545   629  1922 W SurfaceFlinger: Attempting to set client state on removed layer: com.nicolasbrailo.vlcfreemote/com.nicolasbrailo.vlcfreemote.MainActivity#1
11-25 17:41:03.546   629  1922 W SurfaceFlinger: Attempting to set client state on removed layer: Dim Layer for - Task=3176#0
11-25 17:41:03.546   629  1922 W SurfaceFlinger: Attempting to destroy on removed layer: com.nicolasbrailo.vlcfreemote/com.nicolasbrailo.vlcfreemote.MainActivity#1
11-25 17:41:03.546   629  1922 W SurfaceFlinger: Attempting to destroy on removed layer: Dim Layer for - Task=3176#0
11-25 17:41:04.164   629   629 I ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
11-25 17:41:04.169   629   629 I chatty  : uid=1000(system) /system/bin/surfaceflinger identical 2 lines
11-25 17:41:04.170   629   629 I ConfigStore: android::hardware::configstore::V1_0::ISurfaceFlingerConfigs::hasHDRDisplay retrieved: 0
11-25 17:41:04.385   782  2922 I LocSvc_libulp: ulp_msg_process_system_update: systemEvent:5 
```

**Comment by jfcolom** [comment]
Hello!

I'm experiencing the same problem after updating Android (Lineage OS) from 7 to 10. Maybe it has something to do with the fact that now, HTTPS seems to be the default (more information [here](https://stackoverflow.com/questions/51902629/how-to-allow-all-network-connection-types-http-and-https-in-android-9-pie))

Kind regards

**Comment by nicolasbrailo** [comment]
@jfcolom this is great info, thanks! I was able to repro the problem now and will publish an update soon.

**Comment by nicolasbrailo** [comment]
Fixed @ fc0be686506a9758c20ea2c39fbe423b750f0520
I had to update a few things to import to my IDE, so there's a non-zero chance this fix may break other things!

**Comment by marmistrz** [comment]
Thank you!
